tracer/src/Datadog.Trace/HttpHeaderNames.cs

```diff
@@ -52,5 +52,11 @@ public static class HttpHeaderNames
         /// ID of a span.
         /// </summary>
         public const string SpanId = "x-datadog-span-id";
+
+        /// <summary>
+        /// If header is set to "true", the extension will know that the current invocation has failed
+        /// Used in a serverless context only.
+        /// </summary>
+        public const string InvocationError = "x-datadog-invocation-error";
```

Comment:
"Used in a serverless context only." Thanks!

Should we add the same comment to `SpanId = "x-datadog-span-id"` above to avoid confusion with `ParentId = "x-datadog-parent-id"` which is what APM normally uses?

Reply:
✅